admin/app/controllers/authenticated/users/get.js

```diff
@@ -38,6 +41,13 @@ export default class GetController extends Controller {
     }
   }
 
+  _getGARauthenticationMethodId() {
+    const authenticationMethod = this.model.authenticationMethods.filter(
```

Comment:
```suggestion
const authenticationMethods = this.model.authenticationMethods.filter(
```
le filter renvoie un tableau